### PR TITLE
Grab count for all assemblies

### PR DESCRIPTION
**Pull request by tcezard**

### CODE REVIEW

File: eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py [hunk 1]
```diff
@@ -138,12 +138,17 @@ def fill_num_rs_id_for_taxonomy_and_assembly(mongo_source, private_config_xml_fi
 
 
 def get_assembly_list_for_taxonomy(private_config_xml_file, taxonomy):
-    assembly_list = []
+    assembly_list = set()
     with get_metadata_connection_handle("development", private_config_xml_file) as pg_conn:
         query = f'SELECT assembly_accession from evapro.assembly where taxonomy_id = {taxonomy}'
```
Comment (nitin-ebi) [inline eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py:143]: question for my better understanding. 
I used to think **evapro.assembly** is like a master reference table which contains all the assemblies, but that does not seem true now, if there are assemblies present in **eva_progress_tracker.clustering_release_tracker** and not in **evapro.assembly**

Comment (tcezard) [reply eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py:143]: The metadata database capture all assemblies that go through our submission system. So until we started remapping that was true. 
Now that we are remapping there are assemblies that never received submissions that will have submitted and clustered variants.

File: eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py [hunk 2]
```diff
@@ -118,7 +118,7 @@ def fill_in_from_previous_inventory(private_config_xml_file, release_version):
 
 def fill_num_rs_id_for_taxonomy_and_assembly(mongo_source, private_config_xml_file, release_version, taxonomy,
                                              reference_directory):
-    assembly_list = get_assembly_list_for_taxonomy(private_config_xml_file, taxonomy)
+    assembly_list = get_assembly_list_for_taxonomy(private_config_xml_file, taxonomy, release_version)
```
[the remark this answers is not in the document]
Comment (tcezard) [reply eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py:121]: True I forgot to add it to the query.
Changed it

**Review by nitin-ebi — APPROVED** [review]

File: eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py (hunk 1 above)
Comment (nitin-ebi) [reply eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py:143]: Thanks for the clarification

**Review by andresfsilva — APPROVED** [review]

**Pull request merged**